### PR TITLE
Exp 201609

### PR DESCRIPTION
**Pull request by feliciahalim**
Modify Coreutils Makefile and Add a new script to run the experiment.

### CODE REVIEW

File: coreutils/Makefile [hunk 1]
```diff
@@ -118,6 +135,104 @@ sandbox:
...
+%.tracerx2 : 
+	KLEE_COREUTILS_MORE_OPTIONS="--search=dfs -max-subsumption-failure=3 -max-time=7200" OUTPUT_DIR=${CURDIR}/$@ make $*.kleetest
+
+%.tracerx3 : 
+	KLEE_COREUTILS_MORE_OPTIONS="--search=random-path --search=nurs:covnew -max-time=7200" OUTPUT_DIR=${CURDIR}/$@ make $*.kleetest
+
+%.tracerx4 : 
+	KLEE_COREUTILS_MORE_OPTIONS="--search=random-path --search=nurs:covnew -max-subsumption-failure=3 -max-time=7200" OUTPUT_DIR=${CURDIR}/$@ make $*.kleetest
+
+%.kleetest: build sandbox
```
Comment (domainexpert) [inline coreutils/Makefile]: Maybe `%.klee-exp-201609` as a target name is clearer.

File: exp-201609.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (domainexpert) [inline exp-201609.sh]: This loop can be implemented within the `coreutils/Makefile` itself, as a new `exp-201609:` target. In this way, we don't need a separate shell file.

**Review by domainexpert — CHANGES_REQUESTED** [review]
Also, please do not forget to update `README.md` with an instruction on how to run the experiment.

**Comment by feliciahalim** [comment]
@domainexpert Thank you for the feedback. I have updated it accordingly with added `sloccount` in`.klee-exp-201609` target.

File: coreutils/Makefile [hunk 3]
```diff
@@ -273,8 +393,18 @@ sandbox:
 	rm -f ${COREUTILS_GCOV_DIR}/src/*.gcda; 
 
 clean:
-	rm -rf ${KLEE_TARGETS} ${KLEESTP_TARGETS} sandbox sandbox.temps test.env
+	rm -rf ${KLEE_TARGETS} ${KLEE_TARGETS1} ${KLEE_TARGETS2} ${TRACERX_TARGETS1} ${TRACERX_TARGETS2} ${TRACERX_TARGETS3} ${TRACERX_TARGETS4} ${KLEESTP_TARGETS} sandbox sandbox.temps test.env
 
 realclean: clean
 	rm -rf ${COREUTILS_GCOV_DIR} ${COREUTILS_LLVM_DIR} 
 
+exp-201609:
```
Comment (domainexpert) [inline coreutils/Makefile:403]: @feliciahalim Would you please add a comment here explaining this target.

**Review by domainexpert — CHANGES_REQUESTED** [review]

**Comment by domainexpert** [comment]
@feliciahalim Would you please also remove the WIP: from the title of this PR, as this is a PR that is complete.

**Comment by feliciahalim** [comment]
@domainexpert I have updated `coreutils/Makefile` with a comment, updated README.md by giving instructions to run exp-201609 experiment, and also updated this PR title. Thank you.

**Review by domainexpert — APPROVED** [review]

**Comment by domainexpert** [comment]
@feliciahalim Well done! Removing the branch now ...

**Pull request merged**